ivy/functional/frontends/jax/numpy/manipulations.py

```diff
@@ -132,6 +132,10 @@ def atleast_1d(*arys):
 def atleast_2d(*arys):
     return ivy.atleast_2d(*arys)
 
+@to_ivy_arrays_and_back
+def block(arrays):
+    return ivy.block(arrays)
```

Comment:
there is no ivy.block function for you to use directly like this

Reply:
ohh okey I thought that it will work as we use numpy.block(). I'll look into it.